### ISSUE TITLE
empty_line_after_outer_attr false positive with block comments

### ISSUE THREAD

**Issue by tspiteri** [rust-lang-nursery/rust-clippy#2583]
For the following code:
```rust
#[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
/*

*/
pub struct S;
```
clippy 0.0.190 produces the following warning:
```
warning: Found an empty line after an outer attribute. Perhaps you forgot to add a '!' to make it an inner attribute?
 --> src/lib.rs:1:1
  |
1 | / #[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
2 | | /*
3 | |
4 | | */
5 | | pub struct S;
  | |_
  |
  = note: #[warn(empty_line_after_outer_attr)] on by default
  = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.190/index.html#empty_line_after_outer_attr
```

This false positive is not triggered when the attribute is `derive` rather than `cfg`, or if the item is a function rather than a struct.

**Comment by phansch** [comment]
Yea, IIRC, `cfg` is a special attribute. I will have a look, thanks!
